components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/JWTTokenIssuer.java

```diff
@@ -138,8 +138,12 @@ public String accessToken(OAuthAuthzReqMessageContext oAuthAuthzReqMessageContex
     @Override
     public String getAccessTokenHash(String accessToken) throws OAuthSystemException {
         try {
-            JWT parse = JWTParser.parse(accessToken);
-            return parse.getJWTClaimsSet().getJWTID();
+            JWT parsedJwtToken = JWTParser.parse(accessToken);
+            String JTI = parsedJwtToken.getJWTClaimsSet().getJWTID();
```

Comment:
Can we change the variable name to 'jti' or jwtId?

Reply:
fixed